dev-python/capturer/capturer-2.4.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Easily capture stdout/stderr of the current process and subprocesses"
+HOMEPAGE="https://capturer.readthedocs.io/en/latest/
+	https://pypi.org/project/capturer/
+	https://github.com/xolox/python-capturer"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+IUSE="test"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
Btw why are you adding these prefix keywords on these ebuilds?

Reply:
They were already there in the 2.3 ebuild, but I'll get rid of them.

Reply:
It wasn't a complaint, I was just curious.

Reply:
> It wasn't a complaint, I was just curious.

Oops, sorry, should I add it back?

To be honest, I'm not completely sure what `~amd64-linux` would do and how it would be different from just `~amd64`. As I understand it from the wiki, `~amd64-linux` is a subset of `~amd64`, so it doesn't have to be there, right?

Reply:
They are prefix profiles. If the keyword exists, don't drop it when bumping.

Reply:
Alright, I re-added them